.github/workflows/ci.yml

```diff
@@ -493,6 +503,12 @@ jobs:
         name: Integration test setup
         uses: ./.github/actions/integration-test-setup
 
+      # Don't use Chrome for testing (just regular Chrome) until https://bugs.chromium.org/p/chromedriver/issues/detail?id=4533 is resolved
```

Comment:
Nice, can we log an issue for this to get back to it later?

Reply:
https://github.com/keycloak/keycloak/issues/22214